src/Studio/Plugins/Graph/GraphAttributesWidget.cpp

```diff
@@ -31,7 +31,9 @@
 
 
 using namespace Core;
-
+Core::Array<Core::String> ch_data = {}, ch_data_2 = {}, ch_data_3 = {}, ch_data_4 = {}, ch_data_5 = {}, ch_data_6 = {}, ch_data_7 = {}, ch_data_8 = {};
```

Comment:
@cyberjunk can you please double check this?

Reply:
@dioptre  Right now we have not any synthetic fallback maybe it will be available in the coming days.